tilelog/aggregate.py

```diff
@@ -29,28 +30,32 @@
     year,
     month,
     day,
-    hour""".format(regex=TILE_REGEX)
-
+    hour"""
 
 def create_parquet(curs, date):
 
     # Start by checking if any rows are in the table to avoid running twice.
     # This won't catch multiple parallel runs, but will catch running it twice in a row.
-    check_query = """
+    tablename = tilelog.constants.FASTLY_PARQET_LOGS
+    check_query = f"""
 SELECT * FROM {tablename}
 WHERE year = %(year)d
     AND month = %(month)d
     AND day = %(day)d
 LIMIT 1;
-    """.format(tablename=tilelog.constants.FASTLY_PARQET_LOGS)
+    """
     curs.execute(check_query,
                  {"year": date.year, "month": date.month, "day": date.day})
     # curs.rowcount doesn't work for Athena, so iterate through the row. This does nothing
     # if there are no rows in the table for this day, otherwise, it throws an exception.
     for line in curs:
         raise RuntimeError("aggregation queries have already been run for this day")
 
-    insert_query = """
+    columns=SELECT_COLUMNS
```

Comment:
These need spaces around the `=`. Additionally, there's no need for a new variable, just use the right-hand portion in the f-string.

Reply:
Fortunately I could remove most of these errors since I no longer need to create my own variables.